src/sentry/integrations/slack/notify_action.py

```diff
@@ -44,7 +44,22 @@ def clean(self):
         workspace = cleaned_data.get("workspace")
         channel = cleaned_data.get("channel", "")
 
-        channel_prefix, channel_id, timed_out = self.channel_transformer(workspace, channel)
+        try:
+            channel_prefix, channel_id, timed_out = self.channel_transformer(workspace, channel)
+        except IntegrationError:
```

Comment:
@KKaempen Thinking about this a bit more, I think we might want to subclass `IntegrationError` into `DuplicateDisplayNameError` and catch that here. We often raise `IntegrationError` in other places and we don't want to assume the problem an `IntegrationError` is for the display name.

Reply:
Fixed.